tests/unit/Actions.spec.ts

```diff
@@ -55,37 +58,14 @@ const nrData = {
 
 describe('Action button states', () => {
   let wrapper: any
-  let setEntityType: any
-  let setChanged: any
-  let setValidity: any
-  let setEditing: any
 
   beforeAll(async () => {
     // initialize store
-    store.state.stateModel.tombstone.filingType = 'correction'
-    store.state.stateModel.correctionInformation.type = 'CLIENT'
+    store.stateModel.tombstone.filingType = FilingTypes.CORRECTION
+    store.stateModel.correctionInformation.type = CorrectionErrorTypes.CLIENT
 
-    wrapper = shallowMount(Actions, { store, vuetify })
+    wrapper = shallowMount(Actions, { vuetify })
     await Vue.nextTick()
-
-    setEntityType = async (val: string) => {
-      await wrapper.vm.$store.commit('mutateEntityType', val)
-    }
-
-    setChanged = async (val: boolean) => {
-      // set any changed flag
-      await wrapper.vm.$store.commit('mutatePeopleAndRolesChanged', val)
-    }
-
-    setValidity = async (val: boolean) => {
-      // set certify state flag since it's initially false
-      await wrapper.vm.$store.commit('mutateCertifyStateValidity', val)
-    }
-
-    setEditing = async (val: boolean) => {
-      // set any editing flag
-      await wrapper.vm.$store.commit('mutateEditingCompanyName', val)
-    }
```

Comment:
Yes, this was probably wrong since we shouldn't have to await mutations (I think).

However, some actions are async and need to be awaited, fwiw.

Reply:
I think I just rolled this into the actions